frontend/src/slices/articles/actions.ts

```diff
@@ -79,4 +81,38 @@ const getArticle = createAsyncThunk<
   return articleApi.getArticle(id);
 });
 
-export { createArticle, fetchAll, fetchOwn, getArticle, updateArticle };
+const shareArticle = createAsyncThunk<
+  { link: string },
+  { id: string },
+  AsyncThunkConfig
+>(`${sliceName}/share`, (articlePayload, { dispatch, extra }) => {
+  const { articleApi } = extra;
+  void dispatch(
+    appActions.notify({
+      type: NotificationType.SUCCESS,
+      message: 'The sharing link was copied to clipboard',
+    }),
+  );
+
+  return articleApi.share(articlePayload.id);
```

Comment:
Let's notify about success only after response is already given
```suggestion
  const response = await articleApi.share(articlePayload.id);

  void dispatch(
    appActions.notify({
      type: NotificationType.SUCCESS,
      message: 'The sharing link was copied to clipboard',
    }),
  );

  return response;
```